torch/testing/_internal/opinfo/definitions/signal.py

```diff
@@ -292,7 +292,6 @@ def make_signal_windows_opinfo(
         name=name,
         ref=ref if TEST_SCIPY else None,
         dtypes=floating_types(),
-        dtypesIfCUDA=floating_types(),
```

Comment:
FYI, so, so many more of these in common_method invocations. Especially if you filter them properly and realize they are the same args in different orders.